site/content/release-notes.md

```diff
@@ -15,6 +15,7 @@ title = "Release Notes"
 
 * The Device Registry's AMQP endpoints can now be configured with the number of credits they should flow to clients connecting to the endpoints. The default value is 100. See [Device Registry admin guide]({{< relref "admin-guide/device-registry-config.md#service-configuration" >}}) for details.
 * Hono's service components can now be configured to use OpenSSL instead of the JVM's default SSL engine. The [admin guide]({{< relref "admin-guide/secure_communication.md#using-openssl" >}}) describes how to do this.
+* Hono deployment scripts are available under new module named "deploy". These scripts which were under module "example" were moved to "deploy".
```

Comment:
we do not use double quotes to emphasize. please use single back quotes in order to mark as literals.

Reply:
I have updated the description accordingly.